coremain/run.go

```diff
@@ -62,7 +62,7 @@ func Run() {
 	}
 
 	log.SetOutput(os.Stdout)
-	log.SetFlags(log.LstdFlags)
+	log.SetFlags(log.Ldate | log.Ltime | log.LUTC)
```

Comment:
Changing to UTC will surprise everyone, and the date does not say the TZ.
So maybe, while waiting we implement a complete date with TZ in our pkg/log, we can output an info here:
```
	clog.Info("log output - all dates are UTC")
```

Result is:

```
API server listening at: 127.0.0.1:55958
2018/10/29 14:51:55 [INFO] log output - all dates are UTC
.:1053
.:1054
2018/10/29 14:51:55 [INFO] CoreDNS-1.2.5
2018/10/29 14:51:55 [INFO] darwin/amd64, go1.10.2, 
CoreDNS-1.2.5
darwin/amd64, go1.10.2, 
2018/10/29 14:51:55 [INFO] plugin/reload: Running configuration MD5 = 67fbea4124263c10c9c1aa0fb55a538f
```